internal/resource/statefulset.go

```diff
@@ -302,6 +298,12 @@ func (builder *StatefulSetBuilder) podTemplateSpec(annotations, labels map[strin
 				EmptyDir: &corev1.EmptyDirVolumeSource{},
 			},
 		},
+		{
+			Name: "rabbitmq-confd",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
```

Comment:
> Our security context in the template spec ensures that RabbitMQ can read the file, so there is no need to change the ownership in the init container.

Whoa, that's huge. Can this be applied to any of our other initContainer prep? I will try this out.

Reply:
Probably we could apply this to some parts of the init container setup. It **can't** be applied to the Erlang cookie because the Erlang VM expects the cookie to be readable only by the user i.e. `0600` permissions, and given how Kubernetes mounts files (user `root`, group as per security context or `root` if none), we have to copy the cookie and set the expected ownership and permissions.